README.rst

```diff
@@ -40,15 +38,51 @@ iati-registry-record-schema.xsd An application-specific extension
 tests/                          Unit tests for the schemas.
 =============================== ========================================
 
-The tests include a series of short XML documents that should pass or
-file when parsed against the schemas in this distribution.  The shell
-scripts for running the tests rely on a Unix environment with the bash
-shell and the xmllint utility, but the test documents will work with
-any schema-aware XML parser. We will add unit tests as schema
-development continues.
+Dev Installation
+----------------
+
+.. code-block:: bash
+
+  # Create and start a virtual environment
+  $ python3 -m venv pyenv
+  $ source pyenv/bin/activate
+
+  # Install Python package dependencies
+  $ pip install -r requirements-dev.txt
+
+
+Tests
+-----
+
+There are two types of tests:
+
+Legacy tests
+~~~~~~~~~~~~
+
+A series of short XML documents are included in the ``tests/../should-pass``  and ``tests/../should-fail`` directories (and sub-directories). These files test elements and attributes added pre-version 2.03 and offer a simple pass/fail check when parsed against the schemas in this distribution.
+
+These tests can be run using:
+
+.. code-block:: bash
+
+   $ xmllint --schema http://www.w3.org/2001/XMLSchema.xsd --noout *.xsd
+   $ ./tests/run-tests.sh
+
+Tests for elements/attributes added in v2.03
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new test structure was added at version 2.03, with additional test cases added to ``tests/should-pass`` and ``tests/should-fail`` directories (and sub-directories).
```

Comment:
The new directory structure should be described in more detail so that it's clear how to use it.

Reply:
https://github.com/IATI/IATI-Schemas/pull/367/commits/b5d4557928ade1587ef7292d034c77fb8b1355b6